spec/lib/api/v3/work_packages/work_package_schema_representer_spec.rb

```diff
@@ -143,6 +143,26 @@
     end
   end
 
+  describe 'overallCosts' do
+    it_behaves_like 'has basic schema properties' do
+      let(:path) { 'overallCosts' }
+      let(:type) { 'String' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.